config/data_providers.yml

```
@@ -12,3 +12,7 @@ shared:
     name: DGFIP
     logo: dgfip.jpeg
     link: https://api.gouv.fr/producteurs/dgfip
+  ministere_des_armees:
+    name: MinistereDesArmees
```

```suggestion
    name: Ministère Des Armées
```
pourquoi c'est collé ..?

---

faute de frappe...